README.md

```diff
@@ -309,7 +410,7 @@ drive pull -files a1/b2
 drive pull -directories tf1
 ```
 
-## Note: Checksum verification:
+###### Verifying Checksum
```

Comment:
This would contradict an earlier comment you've made and break title consistency.